src/oc_erchef/apps/oc_chef_wm/src/oc_chef_action.erl

```diff
@@ -170,7 +170,13 @@ maybe_add_data(Msg, []) ->
 maybe_add_data(Msg, undefined) ->
     ej:set({<<"data">>}, Msg, {[]});
 maybe_add_data(Msg, FullActionPayload) ->
-    ej:set({<<"data">>}, Msg, FullActionPayload).
+    ej:set({<<"data">>}, Msg, maybe_scrub_password(FullActionPayload)).
+
+maybe_scrub_password(Msg) ->
+    case ej:get({"password"}, Msg) of
+        undefined -> Msg;
+        _ -> ej:set({<<"password">>}, Msg, <<"********">>)
```

Comment:
If you make it 7 '*'s, then you're guaranteed that it's not the password because we enforce a password strength of 8 characters. Or at least we used to somewhere.